src/LazyHydrate.js

```diff
@@ -240,7 +240,18 @@ export default {
     if (this.hydrated) return child;
 
     const tag = this.$el ? this.$el.tagName : `div`;
-    const vnode = h(tag);
+
+    let data = {};
+
+    // When we are not hydrated client side in an SSR app, make sure we proxy
+    // along classes/attrs from the SSR element.  Fixes #34.
+    if (this.$el) {
+      data.attrs = Array.from(this.$el.attributes)
+          .reduce((acc, { name, value }) => Object.assign(acc, { [name]: value }), {}),
+      data.class = Array.from(this.$el.classList);
```

Comment:
This seems to cause a suplicate class issue, so in the reproduction example:

```
        <LazyHydrate :triggerHydration="hydrated" class="lazy-class-outer">
            <div class="lazy-class">
                <button @click="onClick">
                    Click me
                </button>
            </div>
        </LazyHydrate>
```

We end up with the following after mount but before hydration:

```
            <div class="lazy-class-outer lazy-class lazy-class-outer">
```

I think it's probably feasible to filter out the dup class using `this.$vnode.data.staticClass`?

Reply:
Filtered out the `staticClass` in 98afb99